src/odemis/gui/cont/tabs.py

```diff
@@ -515,6 +537,10 @@ def load_data(self, data):
             scont = self._overview_stream_controller.addStream(s, add_to_view=True)
             scont.stream_panel.show_remove_btn(True)
 
```

Comment:
I'd suggest to have a dedicated `load_overview_data()` function, as loading the output of an acquisition will be a little different. 

For now add_to_view=True` means it'll be added to all compatible views, so both top-left and top-right views. It should only be shown on the top-left view. A simple way would be to pass as argument `add_to_view=overview_view`. It'd still allow the user to display the overview stream in the top-right view, but that's maybe nice for the user anyway.

BTW, Mahmoud is going to have to do something very similar with the output of the acquisition, so you might want to synchronize with him.